src/command/stamp/list.ts

```diff
@@ -62,6 +62,14 @@ export class List extends StampCommand implements LeafCommand {
     const limitedStamps = filteredStamps.slice(0, this.limit)
 
     const orderedStamps = this.leastUsed ? limitedStamps.sort((a, b) => a.usage - b.usage) : limitedStamps
-    orderedStamps.forEach(stamp => printStamp(stamp, this.console))
+
+    for (let i = 0; i < orderedStamps.length; i++) {
+      const stamp = orderedStamps[i]
+      printStamp(stamp, this.console)
+
+      if (i !== orderedStamps.length - 1) {
+        this.console.divider()
+      }
+    }
```

Comment:
we could create a similar function as `createKeyValue` just for listing with this divider.
For this particular case I could imagine passing the print hook function (`printStamp`) and the array whose elements needs to be divided.